kahuna/public/js/components/gu-lazy-gallery/gu-lazy-gallery.html

```diff
@@ -0,0 +1,60 @@
+<div class="gallery" gu:lazy-gallery-shortcuts>
+
+    <div class="gallery-item" ng-if="galleryCtrl.item">
+
+        <div class="gallery-item__select gallery-item__select--selected"
+             ng:if="ctrl.imageHasBeenSelected(galleryCtrl.item)">
+            <input type="checkbox"
+                   class="gallery-item__select__checkbox"
+                   id="id-{{::galleryCtrl.item.data.id}}-gallery-item__select__checkbox--select"
+                   checked="true"
+                   ng:click="ctrl.deselect(galleryCtrl.item)"/>
+            <label for="id-{{::galleryCtrl.item.data.id}}-gallery-item__select__checkbox--select"
+                   class="gallery-item__select__checkbox__label">
+                <gr-icon>check_box</gr-icon>
+            </label>
+        </div>
+        <div class="gallery-item__select"
+             ng:if="!ctrl.imageHasBeenSelected(galleryCtrl.item)"
+             ng:class="{ 'gallery-item__select--no-pointer-events': ctrl.inSelectionMode}">
+            <input type="checkbox"
+                   class="gallery-item__select__checkbox"
+                   id="id-{{::galleryCtrl.item.data.id}}-gallery-item__select__checkbox--deselect"
+                   ng:click="ctrl.select(galleryCtrl.item)" />
+            <label for="id-{{::galleryCtrl.item.data.id}}-gallery-item__select__checkbox--deselect"
+                   class="gallery-item__select__checkbox__label">
+                <gr-icon>check_box</gr-icon>
+            </label>
+        </div>
+
+        <div class="gallery-item__preview"
+             ng:class="{ 'gallery-item__preview--seen': ctrl.imageHasBeenSeen(galleryCtrl.item),
+                         'gallery-item__preview--selected': ctrl.imageHasBeenSelected(galleryCtrl.item)}">
+            <ui-preview-image-large class="gallery-item__preview__inner"
+                                    image="galleryCtrl.item"
+                                    selection-mode="inSelectionMode">
+                <button class="mark-as-seen image-action"
+                        title="Mark all images until this point as seen">
+                    <gr-icon>remove_red_eye</gr-icon>
+                </button>
+            </ui-preview-image-large>
+        </div>
+
+    </div>
+
+    <div class="gallery__controls">
+
+        <button class="gallery__control gallery__control--left"
+                ng:click="galleryCtrl.prevItem()">
+            <gr-icon>navigate_before</gr-icon>
+            <span class="visibly-hidden">Previous</span>
```

Comment:
For accesibility reasons, yes. If a screen reader user were to tab to this arrow, they'd have no idea what it did. I could always change it to a `title` attribute instead, but this is also pretty standard.